resources/puppetlabs/lein-ezbake/template/foss/ext/debian/ezbake.init.erb

```diff
@@ -66,7 +66,8 @@ do_start()
     <%= action %>
     <% end -%>
 
-    /opt/puppetlabs/server/apps/${REALNAME}/bin/${REALNAME} start >> /var/log/puppetlabs/${REALNAME}/${REALNAME}-daemon.log 2>&1
+    start-stop-daemon --start $EXTRA_ARGS --quiet --oknodo --pidfile $PIDFILE --chdir $INSTALL_DIR --exec $JAVA_BIN \
```

Comment:
`$EXTRA_ARGS` includes `--background` and `--make-pidfile`, which seem unneeded/confusing with the start changes.